### DISCUSSION TITLE
Graphiql 3.0 how I enable the documentation panel by default?

### DISCUSSION THREAD

**Discussion by younky-yang** [graphql/graphiql#3394] [General] [answered]
Hi, All, 

Now integrated graphiql 3.0 in my project, but I would like to expand the documentation panel by default when it shows up? I checked the props of GraphiQL, but didn't find an option for this? 

Can anyone help on this? 

thanks

**Comment by acao** [comment, marked as answer]
I'm pretty sure this will work offhand. Enjoy!

```ts
<GraphiQL visiblePlugin={"Documentation Explorer"}  />
```

**Reply by younky-yang** [reply]
Thank you so much on this brilliant answer.

**Reply by acao** [reply]
happy to help!
